guides/ruby-gems/gemfury.md

```diff
@@ -0,0 +1,50 @@
+# Gemfury usage
+
+Alternative way to use private gems on multiple projects and add them to dependencies is publishing on hosted repository [gemfury](https://manage.fury.io/dashboard/dvelp).
+
+## Gem publishing
+
+To publish new gem or new version of the gem you need to do following steps:
+
+- Install gemfury gem: `gem install gemfury`
+- Authenticate with your Gemfury login credentials:  `fury login`
+- Build the gem: `gem build <your_gem_name>.gemspec`
+- Upload your package: `fury push <your_gem_name>-<your_gem_version>.gem --as=dvelp`
+
+That’s it!
+
+## Gem installing
+
+To install gem from gemfury repository you need to follow next steps
+
+- Go to https://manage.fury.io/manage/dvelp/tokens and copy your personal access token
+- Configure your bundler with `bundle config gem.fury.io <your_token>`
+- For builds and deployment, you may configure authentication through the environment: `export BUNDLE_GEM__FURY__IO=<your_token>`
+- Add new source to your Gemfile: `source 'https://gem.fury.io/dvelp'`
+- Install your gem
+
+## Configure authentication on CircleCI and heroku
+
+To allow Circle CI and Heroku to install gems from gemfury you simple need to set your auth token to new environment variable: `BUNDLE_GEM__FURY__IO: <your_token>`
+
+## Gem publishing via Cicrle CI
+
+To automate gem publishing process you can add special scenario on Cicrle CI. For this add following to circle.yml file:
```

Comment:
To have the gem automatically published when a new version is created, add the following to your `circle.yml` file: